CMakeLists.txt

```diff
@@ -64,8 +64,8 @@ if (ENABLE_CHECK_HEAVY_BUILDS)
     # Sanitizers are too heavy. Some architectures too.
     if (SANITIZE OR SANITIZE_COVERAGE OR WITH_COVERAGE OR ARCH_RISCV64 OR ARCH_LOONGARCH64)
         # Twice as large
-        set (RLIMIT_DATA 10000000000)
```

Comment:
?

Reply:
build fails on loongarch with 

```
2024-11-15T11:58:29.6461264Z Nov 15 12:58:29 sccache: warning: The server looks like it shut down unexpectedly, compiling locally instead
2024-11-15T11:58:29.6461953Z Nov 15 12:58:29 sccache: Compile terminated by signal 9
```

i suspect that it ooms

Reply:
It's there precisely to prevent us from adding more complex files that explode the compilation times and memory usage

Reply:
Fixed in https://github.com/ClickHouse/ClickHouse/pull/71945

Please remove this modification.